lib/porky_lib/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module PorkyLib
-  VERSION = "0.11.1"
+  VERSION = "0.11.2"
```

Comment:
PorkyLib hasn't been updated since sometime in 2020.

Since Porky and Daffy are gems and used as such by the other repos, the version needs to be bumped and pushed on rubygems CLI for the other repos to receive updates.

Moving forwards, I think any time there are vulnerabilities being patched, we should do a bump.